doc/overview.rst

```diff
@@ -17,7 +17,7 @@ NAV gives you:
 
 * An overview of your deployed network infrastructure inventory, topology and
   address space allocation/utilization.
-* Alerts via e-mail, SMS and/or Jabber when stuff stops working.
+* Alerts via e-mail and/or SMS when stuff stops working.
```

Comment:
Actually, here it would be *fine* to add Slack to the list, since there are the current docs.

Reply:
ack